themes/custom/droopler_theme/templates/layout/page.html.twig

```diff
@@ -150,7 +150,10 @@
         <div class="row clearfix">
           {{ page.breadcrumb }}
           <main{{ main_content_attributes }}>
-            <section class="section">
+            {% if (header_layout == 'header_with_cta') %}
```

Comment:
A może by to zrobić przez attribute i addClass? Wiem, że to trochę overkill na tę chwilę, ale jeśli kiedyś zajdzie potrzeba, żeby dodać kolejną klasę to byłoby w sam raz.